marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.tsx

```diff
@@ -164,68 +169,72 @@ const SelectedFacets: React.FC<Props> = (props) => {
             dateValues.push(item.facet.stringValues[0]);
           }
           return ((unCheckRest(item.constraint, item.facet)) &&
-                    <Button
-                      size="small"
-                      className={styles.facetGreyButton}
-                      key={index}
-                      onClick={() => clearGreyDateFacet()}
-                      data-cy="clear-date-facet"
-                      data-testid="clear-date-facet"
-                    >
-                      {dateValues.join(" ~ ")}
-                      <Icon type="close"/>
-                    </Button>
+            <HCButton
+              size="sm"
+              variant="outline-light"
+              className={styles.facetGreyButton}
+              key={index}
+              onClick={() => clearGreyDateFacet()}
+              data-cy="clear-date-facet"
+              data-testid="clear-date-facet"
+            >
+              {dateValues.join(" ~ ")}
+              <Icon type="close" />
+            </HCButton>
           );
         } else if (item.rangeValues) {
           if (moment(item.rangeValues.lowerBound).isValid() && moment(item.rangeValues.upperBound).isValid()) {
             let dateValues: any = [];
             dateValues.push(item.rangeValues.lowerBound, item.rangeValues.upperBound);
             return ((unCheckRest(item.constraint, item.facet, item.rangeValues)) &&
-                        <Button
-                          size="small"
-                          className={styles.facetGreyButton}
-                          key={index}
-                          onClick={() => clearGreyRangeFacet(item.constraint)}
-                          data-cy={`clear-grey-${item.rangeValues.lowerBound}`}
-                        >
-                          {facetName + ": " + item.rangeValues.lowerBound + " ~ " + item.rangeValues.upperBound}
-                          <Icon type="close"/>
-                        </Button>
+              <HCButton
```

Comment:
The label and close elements of the facet chiclets are misaligned. Fix and review with UX. FYI: @jbelonoj 
<img width="381" alt="chiclet" src="https://user-images.githubusercontent.com/477757/132907417-7709e2fb-4ace-49c9-afb0-c775d6ccffd6.png">

Reply:
Yes, please. Also, make it a little smaller if possible. Thanks! @wooldridge

Reply:
@wooldridge @jbelonoj this the new look
![image](https://user-images.githubusercontent.com/88790827/133124355-c06bf6d3-36cf-4e9f-b145-03077aaf42c7.png)

Reply:
There are some remaining differences:
1. Colors: Text/icon should be lighter, outline should be darker.
2. Icon alignment: Move close icon up a little bit.
![facet-chiclets](https://user-images.githubusercontent.com/477757/133135315-a25bf624-e674-416b-a14a-2ae7aa93c474.png)

Reply:
@wooldridge i updated the stiles for this issue
![image](https://user-images.githubusercontent.com/88790827/133153848-3b164c14-7117-4b7c-ad8f-80f4c5206b3a.png)